rbd.c

```diff
@@ -1022,8 +1022,9 @@ static int tcmu_rbd_open(struct tcmu_device *dev, bool reopen)
 	char *pool, *name, *next_opt;
```

Comment:
Nit: typo in the commit message: `limiation` -> `limitation`

Reply:
Done.